uber/site_sections/summary.py

```diff
@@ -392,3 +392,35 @@ def volunteers_owed_refunds(self, session):
                                                           and a.weighted_hours >= c.HOURS_FOR_REFUND]
             )]
         }
+
+    @csv_file
+    def attendee_birthday_calendar(self, out, session, year=datetime.now(UTC).year):
+        out.writerow(['Subject', 'Start Date', 'Start Time', 'End Date', 'End Time', 'All Day Event', 'Description',
+                      'Location', 'Private'])
+        for person in session.query(Attendee).filter(Attendee.birthdate != None).all():
+            subject = "%s's Birthday" % person.full_name
+            bday = person.birthdate
+            start_date = bday.replace(year=year)
```

Comment:
FYI:  in order to fix #2536, I replaced `dateparser` with `dateutil`. So now you can do:

```python
>>> from dateutil import parser as dateparser
>>> from dateutil.relativedelta import relativedelta
>>> birthday = dateparser.parse('February 29, 1996')
>>> delta_years = datetime.date.today().year - birthday.year
>>> birthday + relativedelta(years=delta_years)
datetime.datetime(2017, 2, 28, 0, 0)
```